src/products/dto/create-product.dto.ts

```
@@ -0,0 +1,72 @@
+  description?: string;
+
+  @IsNumber({ maxDecimalPlaces: 2 })
+  @Min(0)
+  @Transform(({ value }) => parseFloat(value))
+  price: number;
+
+  @IsString()
+  @Length(1, 50)
+  priceUnit: string;
+
+  @IsString()
```

ESLint

[error] Delete `␍`

(prettier/prettier)